StudentsDB/.gitignore

```diff
@@ -3,5 +3,9 @@ bin/
 */bin/*
 target/classes
 */target/classes/*
+<<<<<<< feature/task4-mysql-integration
```

Comment:
Such "<<<<<<<" prefixes to a given line in a file are put automatically by the diff tool when there is a conflict. Seemingly, you had to resolve a conflict here :) Please correct this file and push it again, since in the current version it is broken :)